sklearn/ensemble/_stacking.py

```diff
@@ -119,12 +136,11 @@ def _concatenate_predictions(self, X, predictions):
                 else:
                     X_meta.append(preds)
 
-        self._n_feature_outs = [pred.shape[1] for pred in X_meta]
+        self._n_feature_outs = [np.array(pred).shape[1] for pred in X_meta]
```

Comment:
We should not have to convert to numpy array because they are already all numpy arrays.